yml/dp02_dc2.yaml

```diff
@@ -8962,13 +8975,15 @@ tables:
     description: RMS noise of the sky background.
     mysql:datatype: FLOAT
     fits:tunit: DN
+    ivoa:ucd: stat.rms;instr.det.noise;instr.skyLevel
```

Comment:
`stat.rms;instr.skyLevel` -- it's just the random noise from the sky level, it's not actual instrumental noise